### DISCUSSION TITLE
How do i detect if a Stargate closes with Computercraft?

### DISCUSSION THREAD

**Discussion by masteri212** [Povstalec/StargateJourney#117] [Q&A] [answered]
I'm pretty new to Computercraft and i managed to make a program in which i can dial different stargates, but i cannot figure out how to detect if it closes and the program dosen't close it. Any help would be greatly appreciated.

**Comment by lukaskabc** [comment, marked as answer]
If you want to detect whether the gate has closed, you can use events.  
[Stargate wiki](https://github.com/Povstalec/StargateJourney/wiki/ComputerCraft#events)  
[cc tweaked wiki](https://tweaked.cc/module/os.html#v:pullEvent)
